src/lib/trace.c

```diff
@@ -108,6 +108,16 @@ static void mtrace_event(const char *data, uint32_t length)
 #define _TRACE_EVENT_NTH_PAYLOAD_IMPL(arg_count)			\
 	META_SEQ_FROM_0_TO(arg_count, _TRACE_EVENT_NTH_IMPL_PAYLOAD_STEP)
 
+#ifdef CONFIG_SUECREEK
```

Comment:
CONFIG_TRACE_UART & CONFIG_TRACE_DMA could be more feature-oriented compilation options.